src/main/java/com/feedzai/commons/sql/abstraction/engine/AbstractTranslator.java

```diff
@@ -342,6 +343,34 @@ public String translate(Update u) {
         return join(temp, " ");
     }
 
+    /**
+     * Translates {@link UpdateFrom}.
+     *
+     * @param updateFrom The object to translate.
+     * @return The string representation of the given object.
+     */
+    public String translate(final UpdateFrom updateFrom) {
+        final Expression from = updateFrom.getFrom();
+
+        if (from == null) {
+            return translate((Update) updateFrom);
+        }
+
+        inject(from);
+
+        final String translate = translate((Update) updateFrom);
+        final StringBuilder stringBuilder = new StringBuilder(translate);
+
+        final int where = translate.indexOf("WHERE");
+        if (where == -1) {
+            stringBuilder.append(" FROM ").append(from.translate()).append(";");
```

Comment:
the `;` is not needed and actually should not be used in these JDCB SQL statements

Reply:
Thanks! Going to remove it